bin/local-env/install-wordpress.sh

```diff
@@ -208,6 +208,7 @@ wp post meta add $WEBM_VIDEO_ID web_stories_poster_id $WEBM_VIDEO_POSTER_ID --qu
 wp media import /var/www/html/wp-content/e2e-assets/example-1.jpg --quiet
 wp media import /var/www/html/wp-content/e2e-assets/example-2.jpg --quiet
 wp media import /var/www/html/wp-content/e2e-assets/example-3.png --quiet
+wp media import /var/www/html/wp-content/e2e-assets/OpenSansCondensed-Light.ttf --quiet
```

Comment:
Actually, I just realized that this is not needed.

This adds the file, which already exists on the server, to the WP media library, but we don't need it in the media library.

So we can remove this again.

Sorry for the troubles!

Reply:
No problem -- removed